src/components/select/Option.svelte

```diff
@@ -0,0 +1,160 @@
+{#if visible}
+  <div
+    aria-selected="{isSelected}"
+    class="{classes}"
+    data-option-value="{value}"
+    data-option-label="{label}"
+    on:click="{onClick}"
+    on:mouseenter="{() => {
+      $selectStore.activeValue = value;
+    }}">
+    <div class="ant-select-item-option-content">
+      <slot>{label}</slot>
+    </div>
+    {#if ['multiple', 'tags'].includes($selectStore.mode) && isSelected}
+      <span
+        class="ant-select-item-option-state"
+        unselectable="on"
+        aria-hidden="true"
+        style="user-select: none;">
+        <CheckOutlined />
+      </span>
+    {/if}
+  </div>
+{/if}
+
+<script>
+  import { getContext, onMount, tick } from "svelte";
+  import { CheckOutlined } from "@/components/icons";
```

Comment:
I noticed this here but we should update all components where we use Icons to only load the icon we use.

it should look something like 
```js
  import { default as CheckOutlined } from "@/components/icons/CheckOutlined.svelte";
```